i6_models/util/mask.py

```diff
@@ -0,0 +1,16 @@
+import torch
+
+
+def mask_tensor(tensor: torch.Tensor, seq_len: torch.Tensor) -> torch.Tensor:
+    """
+    Create a "positive" mask for a tensor (boolean true means position is used)
+    on the same device as the tensor.
+
+    :param tensor: [B,T,....]
+    :param seq_len: [B]
+    :return: Mask of [B,T]
+    """
+    seq_len = seq_len.to(device=tensor.device)
```

Comment:
I am not sure if this case should be made within this function.

Reply:
The device handling? So completely removing tensor and having just a seq_len to mask function?

Reply:
you still need the tensor for the shape right? But I feel like this casting of the seq_len should be done beforehand (and at least in my setups usually already is done with the model call. So this might be redundant imho.

Reply:
You could do it without the tensor (using max(seq_len)), I think we can add this as a second function

Reply:
If seq_len is already on the device, this should have not more overhead than one integer comparison

Reply:
is this max then exportable? 
In general if the overhead is low I don't mind and we can leave it like this.